src/components/Avatar/index.tsx

```diff
@@ -99,31 +100,33 @@ const Avatar = forwardRef(
     };
 
     return (
-      <Element
-        {...rest}
-        ref={ref}
-        role={Element === 'div' ? 'img' : undefined}
-        aria-label={ariaLabel}
-        title={cleanName || 'Placeholder Avatar'}
-        className={cc([
-          {
-            'border--focus': Element !== 'div',
-            'color--white': (DARK_COLORS as ReadonlyArray<string>).includes(bgColor),
-          },
-          'fdsAvatar',
-          'alignChild--center--center',
-          `bgColor--${bgColor}`,
-          `fdsAvatar--${size}`,
-          `fdsAvatar--${radius}`,
-        ])}
-      >
-        {imgUrl && (
-          <span className="fdsAvatar-img" style={{ backgroundImage: `url(${imgUrl})` }} />
-        )}
-        {cleanName && grabInitials(cleanName, initialsLength)}
-        {!(cleanName || imgUrl) && PlaceholderIcon && (
-          <PlaceholderIcon customSize={placeholderIconSize()} />
-        )}
+      <Element href={rest.href}>
+        <RadixAvatar.Root
+          {...rest}
+          ref={ref}
+          role={Element === 'div' ? 'img' : undefined}
+          aria-label={ariaLabel}
+          title={cleanName || 'Placeholder Avatar'}
+          className={cc([
+            {
+              'border--focus': Element !== 'div',
+              'color--white': (DARK_COLORS as ReadonlyArray<string>).includes(bgColor),
```

Comment:
can we do this differently so we don't need `as`?

Reply:
this was here from before i worked at CBI - im thinking we leave it as is for now and then when we migrate the component to use Tailwind, well fix it up

Reply:
ah, alright, sounds good to me